CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 - Move 'none' option out of disambiguation place list
 - Move 'back' option out of list
 - Update npm dependencies
+- Display actual search term when no matches in results.
```

Comment:
Any chance of this being updated. See previous https://github.com/nhsuk/connecting-to-services/pull/399#discussion_r153731532